docs/kubernetes/nginx/nginx-ingress.yaml

```diff
@@ -94,7 +94,6 @@ spec:
           - --udp-services-configmap=$(POD_NAMESPACE)/udp-services
           - --annotations-prefix=ingress.kubernetes.io
           - --enable-ssl-passthrough
-          - --ingress-class=tectonic
```

Comment:
There is still a reference to tectonic-custom-error above, should it be?

Please note that I don't know whats going on so well, I'm mainly using generic ideas to review this.

Reply:
Ah yepp but above this, I could not comment on it directly:

One commented line, and one configmap setting.

```
       containers:
         - name: nginx-ingress-lb
           image: quay.io/kubernetes-ingress-controller/nginx-ingress-controller:0.16.2
           args:
           - /nginx-ingress-controller
           - --configmap=$(POD_NAMESPACE)/tectonic-custom-error
           - --default-backend-service=$(POD_NAMESPACE)/default-http-backend
           #- --default-ssl-certificate=tectonic-system/tectonic-ingress-tls-secret
```

Reply:
Those lines indeed have no influence at all if the configmap doesn't exist. But I removed them! I only did a search on `tectonic` in the `mailu` subdirectory, not in the `nginx` directory. This should be solved now. All traces of tectonic are wiped out.

Reply:
They will get removed in the following commit in which I will look into the Ingress Controller manifest as well. For this I need some time to test and make sure it works. I'll get beck here soon!